src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -337,9 +337,9 @@ export interface IInternalEditorOpenOptions extends IInternalEditorTitleControlO
 	readonly preserveWindowOrder?: boolean;
 
 	/**
-	 * Whether to add the editor to the selection or not.
+	 * Whether to add the editor to the current selection or not.
```

Comment:
This is still hard to understand:
- what if the editor opens active but `addToSelection: false`?
- implicitly `addToSelection` is always `true` unless `inactive: true`

Reply:
`addToSelection` means, add to current selection. (Open to better names)
So:
- `addToSelection`: `true`, `active`: `true` -> Opens editor as active and adds it to the existing selection (If 2 were selected before, then now 3 are)
- `addToSelection`: `true`, `active`: `false` -> Opens editor as inactive and adds it to the existing selection (If 2 were selected before, then now 3 are)
- `addToSelection`: `false`, `active`: `true` -> Opens editor as active and creates a new selection with only itself (If 2 were selected before, then now 1 is)
- `addToSelection`: `false`, `active`: `false` -> Opens editor as inactive and does not become selected (If 2 were selected before, then now 2 are)